navigation/ca_move_base/config/lama_params.yaml

```diff
@@ -0,0 +1,23 @@
+# Parameters for LAMA - Localization and Mapping package
+
+# For a full documentation of the parameters used by this package, please check
+# https://github.com/iris-ua/iris_lama_ros
+
+# Initial x position
+initial_pos_x: -3 # Match the create's odom offset
```

Comment:
These 3 parameters are hardcoded for a particular environment. You should set these parameters in the launch file instead based on this: https://github.com/RoboticaUtnFrba/create_autonomy/blob/kinetic-devel/ca_gazebo/launch/create_autorace.launch#L2-L8

Reply:
If you run this, it will fail:

```bash
export LOCALIZATION=lama
export RVIZ=true
export LASER=rplidar
roslaunch ca_gazebo create_stage_1.launch
```

![Screenshot from 2020-06-24 17-39-47](https://user-images.githubusercontent.com/3400230/85625435-d459b600-b641-11ea-9818-90dc864b1004.png)

Reply:
@eborghi10 so I'll need to do it for the house and any other environment available?

Reply:
The solution should be generic so there has to be a way of getting this parameter (remapping them? IDK if that feature exists). If it's too complicated or it can be solved easily, we can leave it as a tech debt, but you should make it written somewhere. A `README` inside the package?

Reply:
Now that I'm seeing your implementation, it can be easily improved (I think), getting the value of `LOCALIZATION` but you should rename the node name to `hector...`.

Reply:
I had renamed the node to `lama`. This switch to use the localization value also involves doing it on the launchfile for every env, right? Note that the parameters don't have the same name for both packages: amcl calls them `initial_pose_<sth>` whereas lama calls them `initial_pos_<sth>`.
I didn't find a way to do a parameter remapping within lama's launchfile.

Reply:
I'm not sure there is a generic way of doing this other than updating each env file or handing the params as arguments for the following launchfiles until it reaches lama and amcl (checking the rest of the envs I see there are some differences so this isn't likely possible)

Reply:
@eborghi10 I went ahead and created a README file as you suggested, with a note. Please let me know wdyt. Thanks!